formidling/src/main/java/no/nav/ung/sak/formidling/vedtak/regler/strategy/EndringInntektReduksjonStrategy.java

```diff
@@ -21,7 +23,18 @@ public EndringInntektReduksjonStrategy(EndringRapportertInntektInnholdBygger end
 
     @Override
     public VedtaksbrevStrategyResultat evaluer(Behandling behandling, LocalDateTimeline<DetaljertResultat> detaljertResultat) {
-        return VedtaksbrevStrategyResultat.medBrev(DokumentMalType.ENDRING_INNTEKT, endringRapportertInntektInnholdBygger, "Automatisk brev ved endring av rapportert inntekt.");
+        boolean harUtførtKontrollerInntekt = behandling.getAksjonspunkterMedTotrinnskontroll().stream()
```

Comment:
Er det nokon grunn til å filtrere på aksjonspunkter med totrinn? Dersom vi på eit tidspunkt finner ut at KONTROLLER_INNTEKT ikkje skal ha totrinn vil det då ikkje gi mulighet for å redigere brev.